server/utils/users/index.test.ts

```diff
@@ -0,0 +1,51 @@
+import { userSummaryListItems } from '.'
+import { userFactory } from '../../testutils/factories'
+
+describe('UserUtils', () => {
+  it('userSummaryListItems', () => {
+    const user = userFactory.build()
+
+    expect(userSummaryListItems(user)).toEqual([
+      {
+        key: {
+          text: 'Name',
+        },
+        value: {
+          text: user.name,
+        },
+      },
+      {
+        key: {
+          text: 'Username',
+        },
+        value: {
+          text: user.deliusUsername,
+        },
+      },
+      {
+        key: {
+          text: 'Email',
+        },
+        value: {
+          text: user?.email ?? 'No email address available',
+        },
+      },
+      {
+        key: {
+          text: 'Phone number',
+        },
+        value: {
+          text: user?.telephoneNumber ?? 'No phone number available',
+        },
+      },
```

Comment:
I feel like we need to test what happens without a user email / phone number in a seperate test, rather than with the logic in the tests here (sure this is just a copy paste error though)

Reply:
oooops